libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -223,20 +210,11 @@ private String determineConfigEndpoint() {
     return COM_CONFIG_ENDPOINT;
   }
 
-  private List<String> extractResponse(Response response) throws IOException {
-    String responseData = response.body().string();
-
-    Gson gson = new Gson();
-    JsonObject jsonObject = gson.fromJson(responseData, JsonObject.class);
-
-    Type listType = new TypeToken<List<String>>() {}.getType();
-    List revokedKeys = gson.fromJson(jsonObject.get("RevokedCertKeys"), listType);
-
-    return revokedKeys;
-  }
+  private void saveTimestamp() {
+    Date date = new Date();
+    SharedPreferences.Editor editor = sharedPreferences.edit();
 
-  private String[] separateUrlSegments(String url) {
-    url = url.replace(HTTPS, EMPTY_STRING);
-    return url.split(BACKSLASH);
+    editor.putLong(MAPBOX_SHARED_PREFERENCE_KEY_BLACKLIST_TIMESTAMP, date.getTime());
+    editor.apply();
```

Comment:
This is done since `apply` is async (writes to memory immediately and schedules write to long term storage) and keeps the process off the main thread. We generally don't use `commit` for this very reason. We can discuss changing that methodology and investigating ways to handle this.